java-frontend/src/main/java/org/sonar/java/cfg/CFG.java

```diff
@@ -700,6 +700,15 @@ private void buildBreakStatement(BreakStatementTree tree) {
       targetBlock = labelsBreakTarget.get(label.name());
     }
     currentBlock = createUnconditionalJump(tree, targetBlock);
+    if(currentBlock.exitBlock != null) {
+      currentBlock.exitBlock = null;
+      Block last = breakTargets.removeLast();
+      targetBlock.successors().clear();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "targetBlock" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2259)